bin/mpi_select_mock_targets

```diff
@@ -92,13 +92,13 @@ if rank == 0:
     keep = list()
     for i, pixnum in enumerate(pixels):
         truthspecfile = mockio.findfile('truth', args.nside, pixnum,
-                                        basedir=args.output_dir)
+                                        basedir=args.output_dir, obscon='dark')
         if not os.path.exists(truthspecfile):
```

Comment:
We should definitely make sure that `mock.io.findfile` matches how the "real" target catalogs are organized, but I thought we already had a `dark`/`bright` organization implemented.

Reply:
There is, but I ran into errors due to a bug on this.  I'd have to refresh my memory by comparing to master.